cognite_toolkit/_cdf_tk/builders/_base.py

```diff
@@ -60,7 +60,11 @@ def build_dir(self) -> Path:
 
     @abstractmethod
     def build(
-        self, source_files: list[BuildSourceFile], module: ModuleLocation, console: Callable[[str], None] | None = None
+        self,
+        source_files: list[BuildSourceFile],
+        module: ModuleLocation,
+        console: Callable[[str], None] | None = None,
+        accept_invalid_files: bool = False,
```

Comment:
Any idea is to instead call it something like this
```suggestion
        validation: Literal["identifier", "full"] = "full",
```

Reply:
Much better! Implemented.